### ISSUE TITLE
Required space in class methods names

### ISSUE THREAD

**Issue by andre487** [ymaps/codestyle#62]
I hava this error with loris codestyle:
```
Missing space before opening round brace at ./examples/config.js :
     4 |
     5 |class Config {
     6 |    constructor() {
-----------------------^
     7 |        this._pagePath = path.join(__dirname, 'base-page.html');
     8 |    }

Missing space before opening round brace at ./examples/config.js :
     8 |    }
     9 |
    10 |    before() {
------------------^
    11 |        return new Promise((resolve, reject) => {
    12 |            fs.readFile(this._pagePath, (err, data) => {


2 code style errors found.
```

**Comment by ikokostya** [comment]
It looks like to https://github.com/jscs-dev/node-jscs/issues/1805

**Comment by ikokostya** [comment]
As quick fix you can try to add this rule in your `.jscsrc` file:

```json
{
  "preset": "loris",
  "requireSpacesInAnonymousFunctionExpression": {
        "beforeOpeningRoundBrace": true,
        "beforeOpeningCurlyBrace": true,
        "allExcept": ["shorthand"]
   }
}
```

**Comment by andre487** [comment]
It works, thanks!

I think this rule should be in the preset

**Comment by ikokostya** [comment]
> I think this rule should be in the preset

I agree

**Comment by ikokostya** [comment]
Fixed in `jscs-preset-loris@1.1.2`

**Comment by andre487** [comment]
:+1:
